src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -318,6 +318,9 @@ private function loadProxyClient(ContainerBuilder $container, XmlFileLoader $loa
         if (isset($config['nginx'])) {
             $this->loadNginx($container, $loader, $config['nginx']);
         }
+        if (isset($config['fastly'])) {
+            $this->LoadFastly($container, $loader, $config['fastly']);
```

Comment:
typo

Reply:
Hi @proeinfo

So did you try this as well? Would be great if that is the case.
Either way, would you be able to use the github suggest feature to suggest fixes to the two things you point out here?

Reply:
Hi @andrerom

I'm working on getting this to work, will post patch once that happens.